modules/microshift-olm-deploy-ops-spec-ns.adoc

```diff
@@ -0,0 +1,222 @@
+<2> The value of `securityContextConfig` must be set to `restricted` for {microshift-short}.
+
+. To verify that the catalog source is applied, check for `READY` state by using the following command:
++
+[source,terminal]
+----
+$ oc describe catalogsources.operators.coreos.com -n openshift-operators operatorhubio-catalog
+----
++
+.Example output
+[source,terminal]
+----
+Name:         operatorhubio-catalog
+Namespace:    openshift-operators
+Labels:       <none>
+Annotations:  <none>
+API Version:  operators.coreos.com/v1alpha1
+Kind:         CatalogSource
+Metadata:
+  Creation Timestamp:  2024-01-30T12:05:14Z
+  Generation:          1
+  Resource Version:    59582
+  UID:                 1260f123-1189-4b68-8153-841a56ee7a5d
+Spec:
+  Display Name:  Community Operators
+  Grpc Pod Config:
+    Security Context Config:  restricted
+  Image:                      quay.io/operatorhubio/catalog:latest
+  Publisher:                  OperatorHub.io
+  Source Type:                grpc
+  Update Strategy:
+    Registry Poll:
+      Interval:  60m
+Status:
+  Connection State:
+    Address:                   operatorhubio-catalog.openshift-operators.svc:50051
+    Last Connect:              2024-01-30T12:05:42Z
+    Last Observed State:       READY
+  Latest Image Registry Poll:  2024-01-30T13:05:14Z
+  Registry Service:
+    Created At:         2024-01-30T12:05:14Z
+    Port:               50051
+    Protocol:           grpc
+    Service Name:       operatorhubio-catalog
+    Service Namespace:  openshift-operators
+Events:                 <none>
+----
+
+. Confirm that the catalog source is running by using the following command:
++
+[source,terminal]
+----
+$ oc get pods -n openshift-operators -l olm.catalogSource=operatorhubio-catalog
+----
++
+.Example output
+[source,terminal]
+----
+NAME                          READY   STATUS    RESTARTS   AGE
+operatorhubio-catalog-5tswm   1/1     Running   0          99m
+----
+
+. Create a Subscription CR configuration file by using the following example YAML:
++
+.Example Subscription custom resource YAML
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: my-cert-manager
+  namespace: openshift-operators
+spec:
+  channel: stable
+  name: cert-manager
+  source: operatorhubio-catalog
+  sourceNamespace: openshift-marketplace <1>
+----
+<1> The global namespace. Subscriptions CRs in any namespace can reference catalogs created in the `openshift-marketplace` namespace.
+
+. Apply the Subscription CR configuration by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f _<my-subscription-cr.yaml>_
+----
++
+.Example output
+[source,terminal]
+----
+subscription.operators.coreos.com/nginx-ok1-1399 created
+----
+
+. Create an operand configuration file by using the following command and example YAML:
+//TODO need an example of deploying an operand to complete this procedure?
```

Comment:
By operand you mean CR for the deployed operator? Like in this case for cert-manager?

Beside that, since we're including commands to verify if something is ready, we might want to add command to see the operator is running, e.g.:

```
$ oc get pods -n openshift-operators
NAME                                                              READY   STATUS      RESTARTS   AGE
cert-manager-7df8994ddb-qnxjw                                     1/1     Running     0          30m
cert-manager-cainjector-5746db8fd7-74njj                          1/1     Running     0          30m
cert-manager-webhook-84d67cfcb6-db7f6                             1/1     Running     0          30m
```

Reply:
Yes, exactly. In Daniel's demo, he has the extra step of applying a CR "operand" after applying the Subscription CR. He applies the specific `Kind: Inifispan` (at about minute 10:30). Do we have one we can add cert-manager for this example?

Reply:
That's definitely going to make your documentation rot. We don't have a 1:1 relationship between openshift releases and the stuff that goes into the catalogs. Operators can add breaking changes to the latest version whenever they want. I would avoid doing this, as you're essentially documenting the cert-manager operator at that point

Reply:
@kevinrizza Do we need to note that the user can then install whatever operand (generic narrative step), or do we just leave it there?